cmd/devdata/trb_request.go

```diff
@@ -715,7 +715,16 @@ func (s *seederConfig) addTRBRequest(ctx context.Context, rType models.TRBReques
 	if err != nil {
 		return nil, err
 	}
-
+	attendee, err := s.store.GetAttendeeByEUAIDAndTRBID(ctx, trb.CreatedBy, trb.ID)
+	if err != nil {
+		return nil, err
+	}
+	componentStr := "Center for Medicare (CM)"
+	attendee.Component = &componentStr
```

Comment:
```suggestion
	attendee.Component = helpers.PointerTo("Center for Medicare (CM)")
```

Reply:
As you wish: [135ebf8](https://github.com/CMS-Enterprise/easi-app/pull/2705/commits/135ebf8d00a6c2024261ef008968e4991f51cb57)